bochs/bx_debug/parser.y

```diff
@@ -1158,6 +1173,11 @@ help_command:
          dbg_printf("ldsym [global] <filename> [offset] - load symbols from file\n");
          free($1);free($2);
        }
+     | BX_TOKEN_HELP BX_TOKEN_SET_MAGIC_BREAK_POINTS '\n'
+       {
+         dbg_printf("setmagicbps \"%%cx %%dx %%bx %%sp %%bp %%si %%di\" - changes magic breakpoints. You can specify multiple at once. Using the setmagicbps command without any arguments will disable all of them\n");
```

Comment:
most of the debugger use register without %%
so better to be setmagicbps cx dx
also - how about clearing them ?
I think clearmagicbps needed too